<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1255">
<meta name=ProgId content=PowerPoint.Slide>
<meta name=Generator content="Microsoft PowerPoint 9">
<link id=Main-File rel=Main-File href="../class8.htm">
<link rel=Preview href=preview.wmf>

<script>
<!--
if( window.name != "PPTOtl" )
  parent.location.href = "../class8.htm";
//-->
</script>
<meta name=Robots content=NoIndex>
</head>

<body bgcolor=black text=white link=white vlink=white alink=white>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0420.htm');">Unit 6: Regulatory
Circuits</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0432.htm');">Gene
Expression:<br>
From DNA to Protein</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0421.htm');">Gene Expression:
<br>
From DNA to Protein</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width=20 nowrap></td>
  <td width=20 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=3><font face=Arial size=2>We will shift our level of abstraction</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=3><font face=Arial size=2>Before, we treated each DNA, RNA or
  amino acid monomer as our essential process</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=3><font face=Arial size=2>Now, we will treat functional units as
  essential processes</font></td>
 </tr>
 <tr>
  <td colspan=3></td>
  <td colspan=1><font face=Arial size=2>Genes</font></td>
 </tr>
 <tr>
  <td colspan=3></td>
  <td colspan=1><font face=Arial size=2>RNA molecules</font></td>
 </tr>
 <tr>
  <td colspan=3></td>
  <td colspan=1><font face=Arial size=2>Proteins</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0422.htm');">Multi-Step Regulation
<br>
of Gene Expression</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0423.htm');">Two Views of Regulation</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The biochemical view: What are the
  specific biochemical events that regulate gene expression? E.g. how is
  transcription regulated?</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The logical view: What is the
  functional outcome of a regulatory event: up-regulation (activation) or
  down-regulation (inhibition)?</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0433.htm');">Basal Transcription</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0424.htm');">Regulation of
Transcription: Biochemical View</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0434.htm');">Transcription Factors</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Most transcription factors are
  DNA-binding proteins</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>They recognize and bind regulatory DNA
  motifs: relatively specific short sequences of DNA</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>These regulatory sequences may be in
  the vicinity of the promoter (RNAPII binding site) or at a distance
  (enhancers)</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0435.htm');">Transcription Factors</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Transcription factors modulate basal
  transcription levels, either activating or repressing it</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>In many cases this is mediated by
  facilitating (positive control) or hindering (negative control) the assembly
  of the general transcription complex</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The ability of the regulatory protein
  to bind to DNA is often regulated as well</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0446.htm');">The Principles:
<br>
Prokaryotic Regulation of Transcription</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0440.htm');">Example: The Tryptophan
Repressor</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0441.htm');">Activators and Repressors</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0442.htm');">Activator and Repressor</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0445.htm');">Activator and Repressor</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0444.htm');">Multiple
Controls:<br>
The lac Operon “logic”</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0447.htm');">Regulation At a Distance:
Enhances</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0436.htm');">Eukaryotic Transcription</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Most prokaryotic genes are controlled
  by only one or two gene regulatory proteins</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The regulation of higher eukaryotic
  genes is much more complex, where tens of transcriptional activators and
  repressors may bind to a large regulatory region</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>For example, the control region of the eve
  gene of the fruit fly (Drosophila) is 20,000 nt long, and has binding sites
  for over 20 proteins</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0448.htm');">Gene Control Region</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0449.htm');">Eukaryotic Activators</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0450.htm');">Eukaryotic Repressors</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0437.htm');">Combinatorial Regulation</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>This combination of multiple regulatory
  sites (and proteins) for a single gene allows fine-tuning of transcription
  and its levels</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The action of many transcription
  factors depends on the formation of the correct complex at the regulatory
  site.</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>For example, some factors do not bind
  to DNA directly but are recruited by binding to other DNA binding proteins.</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The same protein may serve as an
  activator or repressor, depending on the context in which it is present</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0451.htm');">Context-Dependent
Function</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0452.htm');">Combinatorial Regulation</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0438.htm');">Combinatorial Regulation</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width=20 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>The activity of a gene regulatory
  protein can itself be regulated, e.g.</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Transcriptionally – control over the
  quantity of proteins</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Post-translationally – control over the
  quantity of active (binding-enabled) protein in the nucleus</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>This leads to the formation of complex regulatory
  circuits</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0439.htm');">Regulating regulatory
proteins</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0443.htm');">The lac Operon “logic”</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0453.htm');">The Lysis/lysogeny
logic<br>
(Shapiro and McAdams, 1995; Arkin et al 1998)</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0454.htm');">The Cell Cycle “logic”</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0455.htm');">The Cell Cycle Logic
(cont)</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0456.htm');">Modeling Approaches</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Logical – treat as Boolean (AND, OR,
  NOT etc) system</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Biochemical – model biochemical events
  as accurately as possible</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Hybrid – Abstract some of the
  biochemical details away while modeling others in detail</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0427.htm');">Example: The Circadian
Clock</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Many organisms (including humans),<span
  style="mso-spacerun: yes">&nbsp; </span>employ different “programs” in light
  and dark conditions</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The switch between the two programs is
  the result of the entrainment of an internal oscillator by external
  conditions (light and dark)</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The internal oscillator is termed the circadian
  clock</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0458.htm');">The Circadian clock</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Circadian clock can be found in many
  organisms including cyanobacteria, fungi, flies, mouse and humans</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>They may have evolved more than once in
  evolution</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>While they differ in the biochemical
  detail, it appears there is an underlying “design principle” in their
  regulatory logic</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>This logic allows faithful intrinsic
  oscillation of these molecular clocks</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0459.htm');">The Circadian Clock</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0460.htm');">The Circadian Model
<br>
(Barkai and Leibler, 2000)</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0461.htm');">The Circadian Model:
Dependency on Differential Rates</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width=20 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>Oscillations will be achieved only if
  there is a certain “imbalance” in the rates in the system</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>“A” related processes are fast (basal
  and activated transcription, translation, and RNA and protein degradation)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>“R” related processes are significantly
  slower (e.g. almost no basal transcription)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>A-R binding is extremely fast</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>A hybrid approach: A “logical” model is
  not enough, but a full biochemical one is currently impossible</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0462.htm');">The Circadian Clock:
System</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>-language(psifcp).<br>
  -include(rates).<br>
  global(pA(R1),bA(R2),transcribeA(R13),utrA(R3),
  degmA(R4),degpA(R12),pR(R5),rbs(R6),bR(R8),
  transcribeR(R14),utrR(R9),degmR(R10),degpR(R11)).<br>
  <br>
  System::=<br>
  <span
  style="mso-spacerun: yes">&nbsp;&nbsp; </span>activator#A_GENE | <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>repressor#R_GENE
  |<span style="mso-spacerun: yes">&nbsp;&nbsp; </span><br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>timer#Timer(transcribeA)
  | <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>timer#Timer(transcribeR)
  |<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;
  </span>machineries#BASAL_TRANSCRIPTION | <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;
  </span>machineries#BASAL_TRANSLATION |<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>machineries#RNA_DEGRADATION
  |<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;
  </span>machineries#PROTEIN_DEGRADATION .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0463.htm');">The Circadian Clock:
<br>
“Gene A”, “RNA A”</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>-language(psifcp).<br>
  -include(rates).<br>
  global(pA(R1),bA(R2),transcribeA(R13),utrA(R3),<br>
  degmA(R4),degpA(R12),pR(R5),rbs(R6)).<br>
  <br>
  A_GENE::=
  &lt;&lt; PROMOTED_A + BASAL_A .<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>PROMOTED_A::= <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>pA ?
  {unbind_A} , ACTIVATED_TRANSCRIPTION_A(unbind_A) .<br>
  <span
  style="mso-spacerun: yes">&nbsp;&nbsp; </span>BASAL_A::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>bA ? [] ,
  A_GENE | A_RNA .<br>
  <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;
  </span>ACTIVATED_TRANSCRIPTION_A(unbind)::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>transcribeA
  ? [] , A_RNA | ACTIVATED_TRANSCRIPTION_A ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>unbind<span
  style="mso-spacerun: yes">&nbsp;&nbsp; </span>? [] , A_GENE &gt;&gt; .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>A_RNA::= &lt;&lt; TRANSLATION_A +
  DEGRADATION_mA .<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>TRANSLATION_A::= utrA ?
  [] , A_RNA | A_PROTEIN<span style="mso-spacerun: yes">&nbsp; </span>.<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>DEGRADATION_mA::= degmA ?
  [] , true &gt;&gt; .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0473.htm');">Syntax Note: Summing
Processes</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Processes with a guarded choice
  construct are called normal processes</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Such processes may be summed together,
  to create a third process, with a combined choice construct, e.g.</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0472.htm');">The Circadian Clock:
“Protein A”</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>A_PROTEIN::=</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2><span style="mso-spacerun: yes">&nbsp;
  </span>&lt;&lt; release_AR(infinite),unbind_A_pA(R7a),unbind_A_pR(R7b).</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2><span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>PROMOTION_AR + BINDING_R + DEGRADATION_A
  .<br>
  <br>
  PROMOTION_AR::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>pA ! {unbind_A_pA}
  , unbind_A_pA ! [] , A_PROTEIN ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>pR ! {unbind_A_pR}
  , unbind_A_pR ! [] , A_PROTEIN .<br>
  <br>
  BINDING_R::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>rbs ! {release_AR}
  , BOUND_A_PROTEIN .<br>
  <br>
  BOUND_A_PROTEIN::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;
  </span>degpA ? [] , release_AR ! [] , true ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>release_AR ? [] ,
  A_PROTEIN .<br>
  <br>
  DEGRADATION_A::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>degpA ? [] , true
  &gt;&gt; .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0474.htm');">The Circadian Clock:
<br>
“Gene R”, “RNA R”</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>-language(psifcp).<br>
  -include(rates).<br>
 
  global(bR(R8),transcribeR(R14),utrR(R9),degmR(R10),degpR(R11),pR(R5),rbs(R6)).<br>
  <br>
  R_GENE::=
  &lt;&lt; PROMOTED_R + BASAL_R .<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>PROMOTED_R::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>pR ?
  {unbind_A} , ACTIVATED_TRANSCRIPTION_R(unbind_A) .<br>
  <span
  style="mso-spacerun: yes">&nbsp;&nbsp; </span>BASAL_R::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>bR ?
  [] , R_GENE | R_RNA .<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;
  </span>ACTIVATED_TRANSCRIPTION_R(unbind)::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>transcribeR ? [] , R_RNA | ACTIVATED_TRANSCRIPTION_R ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>unbind<span
  style="mso-spacerun: yes">&nbsp;&nbsp; </span>? [] , R_GENE &gt;&gt; .<br>
  <br>
  R_RNA::=
  &lt;&lt; TRANSLATION_R + DEGRADATION_mR .<br>
  TRANSLATION_R::= utrR ? [] , R_RNA | R_PROTEIN<span style="mso-spacerun:
  yes">&nbsp; </span>.<br>
  DEGRADATION_mR::= degmR ? [] , true &gt;&gt; .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0464.htm');">The Circadian Clock:
“Protein R”</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>R_PROTEIN::= <br>
  <br>
  &lt;&lt; BINDING_A + DEGRADATION_R .<br>
  <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>BINDING_A::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>rbs ?
  {release_A} , BOUND_R_PROTEIN(release_A) .<br>
  <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;
  </span>BOUND_R_PROTEIN(release)::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>release ? []<span style="mso-spacerun: yes">&nbsp; </span>, R_PROTEIN ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>degpR ? [] , release ! [] , true .<br>
  <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>DEGRADATION_R::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>degpR ? [] , true &gt;&gt; .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0465.htm');">The Circadian Clock:
Machineries</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>-language(psifcp).<br>
  -include(rates).<br>
  global(bA(R2),bR(R8),utrA(R3),utrR(R9),degmA(R4),degmR(R10), degpA(R12),
  degpR(R11)).<br>
  <br>
  BASAL_TRANSCRIPTION::= <br>
  <span
  style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>bA ! [] ,
  BASAL_TRANSCRIPTION ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>bR ! [] ,
  BASAL_TRANSCRIPTION .<br>
  <br>
  BASAL_TRANSLATION::=<br>
  <span
  style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>utrA ! [] ,
  BASAL_TRANSLATION ;</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2><span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp; </span>utrR ! [] , BASAL_TRANSLATION .<br>
  <br>
  RNA_DEGRADATION::=<br>
  <span
  style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>degmA ! [] ,
  RNA_DEGRADATION ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>degmR ! [] ,
  RNA_DEGRADATION .<br>
  <br>
  PROTEIN_DEGRADATION::=<br>
  <span
  style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>degpA ! [] ,
  PROTEIN_DEGRADATION ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>degpR ! [] ,
  PROTEIN_DEGRADATION .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0466.htm');">The Circadian Clock:
Rates</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0467.htm');">Basal Transcription</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0469.htm');">Translation and RNA
degradation</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0476.htm');">Protein A and R</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0468.htm');">Activated Transcription</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0478.htm');">Activated Transcription</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0470.htm');">A-R Binding</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0477.htm');">The Circadian Model
<br>
(Barkai and Leibler, 2000)</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0475.htm');">Circadian Clock: Results</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0457.htm');">Exercise 7</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width=20 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>Modify the circadian clock program in
  the following ways:</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>(a) Run the same system but with the
  same rates for the transcription of R and A (basal=2; activated=40) (1000
  units)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>(b) Abolish the negative feedback of R
  (600 units) <br>
  (no binding between A and R)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>(c) Abolish the positive feedback of A
  on its own transcription (no binding of A to its promoter) (2000 units)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>(d) Abolish the positive feedback of A
  completely <br>
  (no binding of A to any promoter) (1000 units)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>(e) The R repression mechanism is
  mediated by competing with A on the promoter (rather than by binding the A
  protein). R’s binding rate to pA and pR is 100. Its unbinding rate is 10 (800
  units)</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0479.htm');">Exercise 7</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>In each case, run the clock program for
  the noted time units, and plot (scale 1) the behavior of the protein and RNA
  molecules of each species (note: for the total protein you must sum the free
  and bound processes).</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Submit code and plots.</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Explain the difference in the results
  between these runs and the “original” one. If A control “light genes” and R
  controls “day genes” what is the effect of each of these changes on the
  organism?</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Note: these runs take significantly
  longer than previous programs. Submission in 3 weeks.</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0431.htm');">Biochemical Model:
Arkin’s Stochastic Model</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>A detailed stochastic model of the
  kinetics of transcription and translation and degradation (in prokaryotes)</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Includes initiation, propagation and
  termination stages in each of these processes</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Was used as an underlying biochemical
  model of the transcription, translation and degradation events in the
  lysis/lysogeny circuit</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0410.htm');">References</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>General Introduction: Alberts et al.
  (1994) Molecular Biology of the Cell, Chapter 9, pp.401-433</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The Circadian Clock: Barkai and Leibler
  (2000) Circadian clocks limited by noise. Nature 403:267-268</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Stochastic mechanism of gene
  expression: McAdams and Arkin (1997) Stochastic mechanisms in gene
  expression. PNAS 94:814-819</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Lysis/Lysogeny: Arkin et al (1998)
  Stochastic kinetic analysis of developmental pathway bifurcation in phage
  lambda infected Escherichia coli.<span style="mso-spacerun: yes">&nbsp;
  </span>Genetics 149: 1633-1648</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Cell cycle: Kohn (1999) Molecular
  interaction map of the mammalian cell cycle control and DNA repair systems</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The tryptophan operon: Santillian and
  Mackey (2001) Dynamic regulation of the tryptophan operon: A modeling study
  and comparison with experimental data. PNAS 98: 1364-1369</font></td>
 </tr>
</table>

</p>

</body>

</html>
