<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1255">
<meta name=ProgId content=PowerPoint.Slide>
<meta name=Generator content="Microsoft PowerPoint 9">
<link id=Main-File rel=Main-File href="../class9.htm">
<link rel=Preview href=preview.wmf>

<script>
<!--
if( window.name != "PPTOtl" )
  parent.location.href = "../class9.htm";
//-->
</script>
<meta name=Robots content=NoIndex>
</head>

<body bgcolor=black text=white link=white vlink=white alink=white>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0420.htm');">Unit 7: Signal
Transduction</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0422.htm');">Multi-Step Regulation
<br>
of Gene Expression</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0457.htm');">Signal Transduction
Pathways</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Pathways of molecular interactions that
  provide communication between the<br>
  cell membrane and intracellular
  endpoints, leading to some change in the cell</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0472.htm');">Major themes in ST</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The “internal complexity” of each
  interaction</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The combinatorial nature of each
  component molecule (may receive and send multiple signals)</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The integration of pathways and
  networks</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0462.htm');">Signal source</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>A signaling cell produces a<span
  style="mso-spacerun: yes">&nbsp; </span>particular particular type of signal
  molecule</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>This is detected in another target
  cell, by means of a receptor protein, which recognizes and responds
  specifically to its ligand</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>We distinguish between Endocrine,
  paracrine and autocrine signaling. The latter often occurs in a population of
  homogenous cells.</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Each cell responds to a limited set of
  signals, and in a specific way</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0469.htm');">Signaling<span
style="mso-spacerun: yes">&nbsp; </span>Molecule</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The signal molecule is often secreted
  from the<span style="mso-spacerun: yes">&nbsp; </span>signaling cell to the
  extracellular space</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>In some cases the signaling molecule is
  bound to the cell surface of the signaling cell. Sometimes, a signal in both
  cells will be initiated by such an event.</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0470.htm');">Receptors</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Cell surface receptors detect
  hydrophilic ligands that do not enter the cell</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Alternatively, a small hydrophobic
  ligand (e.g. steroids) may cross the membrane, and bind to an intracellular
  receptor</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Cells may also be linked through a gap
  junction, sharing small intracellular signaling molecules</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0471.htm');">Cell Surface Receptors</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Ion channel linked: Binding of ligand
  causes channel to open or close</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>G-protein linked:<br>
  Binding of ligand activates
  a G-protein which will activate a separate enzyme or ion channel</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Enzyme linked receptor: Binding of
  ligand activates an enzyme domain on the receptor itself or on an associated
  molecule</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0473.htm');">Intracellular receptors</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Small hydrophobic signaling molecules,
  such as steroids, can cross the cell membrane (e.g. estrogen, vitamin D,
  thyroid hormone, retinoic acid) and bind to intracellular receptors</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The hormone-receptor complex has an exposed
  DNA binding site and can activate transcription directly (or, more typically
  as a homo- or hetero-dimer)</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>This usually initiates a cascade of
  transcription events</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0474.htm');">Regulating proteins</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0458.htm');">Molecular Interactions</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width=20 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>Protein-protein interactions</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Binding or unbinding (formation or
  breaking of complex)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Covalent modification:</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>	phosphorylation (tyr, thr, ser)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Conformation changes</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Translocation</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Targeting for degradation</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>Small molecule regulated events</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Binding or unbinding, resulting in
  conformation change: Steroid ligand, nucleotide binding</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Production of second messengers (e.g.
  Ca+2)</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0475.htm');">Covalent and non-covalent
association of phosphate groups</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width=20 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>The association (or absence) of a
  phosphate group with a protein may affect its capability to interact or its
  activity</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Activate an enzymatic domain by
  conformation change</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Enable or disable binding by structural
  change in binding site</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Affect binding/unbinding of complex and
  release of “active form” of a G-protein</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>Both the covalent and non-covalent
  modifications are reversible, and so are their effects.</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0476.htm');">Second messengers</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>In many pathways, enzymes are activated
  which catalyze the formation of a large quantity of small molecules</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>These second messengers broadcast the
  signal by diffusing widely to act on target proteins in various parts of the
  cell</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>This may often result in the release of
  other second messengers</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0477.htm');">Multi-state regulation of
a single protein</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0478.htm');">Integration of Signals</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0479.htm');">Insulation by complex
formation</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The same signaling molecule may
  participate in more than one pathway</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>In such cases, it is sometimes
  insulated from some of its potential inputs and outputs and sequestered (with
  specific up- and downstream counterparts) by a specific scaffold molecule</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0480.htm');">Amplification</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0481.htm');">Intracellular target</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Determining the “end” of a signaling
  pathway is often difficult</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>For example, after transcription, a
  phosphatase may be synthesized that dephosphorylates one of the enzymes in
  the pathway</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>One approach is to consider an event
  that is “biochemically different” (e.g. transcription, metabolism) as the
  intracellular target</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0459.htm');">Intracellular Endpoint</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width=20 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>Three major molecular targets</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Regulation of gene expression (e.g.
  activate a transcription factor and translocate it to the nucleus)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Changes in the cytoskeleton (e.g.
  induce movement or reorganization of cell structure)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Affect metabolic pathways</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>Many critical processes can occur in
  response to external signals, without any new synthesis of RNA or proteins.
  The most well known one is “cell suicide”, termed apoptosis</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0460.htm');">Change in the cell</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>An animal cell depends on multiple
  extracellular signals</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Multiple signals are required to
  survive, additional to divide and still others to differentiate</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>When deprived of appropriate signals
  most cells undergo apoptosis</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0482.htm');">Change in the cell</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The same signal molecule can induce
  different responses in different target cells, which express different
  receptors or signaling molecules</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>For example, the neurotransmitter
  acetylcholine induces contraction in skeletal muscle cells, relaxation in
  heart muscle cells and secretion in salivary gland cells</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0483.htm');">Slide 22</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0423.htm');">Two Views of Signaling</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The biochemical view: What are the
  specific biochemical events that mediate signals?</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The logical view: Is a signal
  activatory or inhibitory?</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0463.htm');">The RTK-MAPK pathway</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0484.htm');">The RTK-MAPK pathway</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0465.htm');">Receptor-Ligand Binding</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>A dimeric ligand protein is formed by
  di-sulfide bonds between two identical protein monomers</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The ligand has two identical receptor
  binding sites and can cross link two adjacent receptors upon their binding</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>This initiates the intracellular
  signaling<span style="mso-spacerun: yes">&nbsp; </span>process</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>We assume that ligand-receptor binding
  is irreversible</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0485.htm');">Receptor Activation</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The cytoplasmic domain of the receptor
  has intrinsic kinase activity</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Upon dimerization each receptor cross
  phosphorylates a specific tyrosine residue on its counterpart, which fully
  activates its kinase</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Then, each kinase autophosphorylates
  additional tyrosine residues on it own cytoplasmic part</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0491.htm');">Ligand</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>global(ligand_bind,dummy).<br>
  <br>
  LIGAND::=</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>&lt;&lt; ligand .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>FREE_BD | FREE_BD .<br>
  <br>
  FREE_BD::= <br>
  	ligand_bind ! {ligand} , BOUND_BD .<br>
  	<br>
  BOUND_BD::= <br>
  	dummy ? [] , true &gt;&gt; .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0492.htm');">Receptor (Extracellular
part)</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>global(ligand_bind,tyr,p_tyr,met,atp,dummy).<br>
  <br>
  RTK(env)::=</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>&lt;&lt; backbone_extra,
  backbone_intra1, backbone_intra2,<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>backbone_intra3,
  tyr1162, atp_bs,sh2_tyr,sh2_tyr1 .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2><span style="mso-spacerun:
  yes">&nbsp;&nbsp; </span>EXTRACELLULAR | TRANSMEMBRANAL | INTRACELLULAR
  .<br>
  <br>
  EXTRACELLULAR::= <br>
  	ligand_bind ? {lig} , backbone_extra ! {lig} ,<span style="mso-spacerun:
  yes">&nbsp;&nbsp; </span><br>
  <span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>BOUND_EXTRACELLULAR .<br>
  <br>
  BOUND_EXTRACELLULAR::=<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>dummy
  ? [] , true .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0493.htm');">Ligand-Receptor binding</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0494.htm');">Receptor (Transmembranal)</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>TRANSMEMBRANAL::=</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>&lt;&lt; cross_receptor .<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>backbone_extra ?
  {cross_lig} ,<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>&lt;&lt; cross_lig
  ! {tyr1162, cross_receptor} ,</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>	<span style="mso-spacerun: yes">&nbsp;
  </span>cross_receptor ? {cross_tyr} , <br>
  <span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>backbone_intra1 ! {cross_tyr} , 		<span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>RTK_DIMERIZED ;<br>
  	 cross_lig ? {cross_tyr, cross_rec} , 		<span style="mso-spacerun:
  yes">&nbsp; </span>cross_rec ! {tyr1162} , <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>backbone_intra1 ! {cross_tyr} , 			 RTK_DIMERIZED &gt;&gt; .<br>
  <br>
  RTK_DIMERIZED:-<br>
  	dummy ? [] | true &gt;&gt; .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0495.htm');">Receptor dimerization</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0499.htm');">Receptor dimerization</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0519.htm');">Receptor Activation</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The cytoplasmic domain of the receptor
  has intrinsic kinase activity</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Upon dimerization each receptor cross
  phosphorylates a specific tyrosine residue on its counterpart, which fully
  activates its kinase</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Then, each kinase autophosphorylates
  additional tyrosine residues on it own cytoplasmic part</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0504.htm');">Location and Chemical
complementarity</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width=20 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>For one receptor to phosphorylate
  another (or itself) the two must share</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Common complex (private channel)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Chemical complementarity (global
  channel)</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>This creates a modeling difficulty,
  since we cannot match two channels simultaneously</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>One option is to use a match construct</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>First communicate on the private
  channel and send a global channel name (bind)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Then, match the global channels by
  comparing them (react)</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>If the second match does not work the
  counterparts unbind (similar to a competitive inhibitor)</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>An simpler alternative is to use only
  the private channels, but this may create an “illegal” situation where the
  kinase phosphorylates something it shouldn’t</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0500.htm');">Receptor (Cytoplasmic)</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>INTRACELLULAR::=<br>
  	RTK_SH_BS(tyr,met) | RTK_KINASE_CORE .<br>
  <br>
  RTK_KINASE_CORE::=<br>
  	RTK_KINASE_SITE | <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>RTK_REGULATORY_SITE(tyr)
  |<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>RTK_ATP_BS<span
  style="mso-spacerun: yes">&nbsp;&nbsp; </span>.</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0501.htm');">RTK Kinase –
Phosphorylation – Option I</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>RTK_KINASE_SITE::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>CROSS_PHOSPHORYLATE +
  FULL_PHOSPHORYLATE .<br>
  <br>
  CROSS_PHOSPHORYLATE::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra1 ?
  {cross_motif} , cross_motif ? {cross_res} , <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp; </span>&lt;&lt; cross_res=?=tyr
  , cross_motif ! {p_tyr} , RTK_KINASE_SITE;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>otherwise , cross_motif ! {cross_res} , RTK_KINASE_SITE &gt;&gt;.
  <br>
  FULL_PHOSPHORYLATE::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra3 ? [] ,
  ACTIVE_FULL .<br>
  <br>
  ACTIVE_FULL::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra2 ? {cross_motif}
  , cross_motif ? {cross_res} , <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>&lt;&lt;
  cross_res=?=tyr<span style="mso-spacerun: yes">&nbsp; </span>, cross_motif !
  {p_tyr} , ACTIVE_FULL ;<br>
  <span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>otherwise ,
  cross_motif ! {cross_res} , ACTIVE_FULL &gt;&gt; ;<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra3 ? [] ,
  RTK_KINASE_SITE .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0502.htm');">RTK Kinase –
Regulation<span style="mso-spacerun: yes">&nbsp; </span>- Option I</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>RTK_REGULATORY_SITE(res)::=<br>
  	tyr1162 ! {res} , tyr1162 ? {res1} ,</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2><span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>&lt;&lt; res1 =?= res<span
  style="mso-spacerun: yes">&nbsp; </span>, RTK_REGULATORY_SITE(res1) ;<br>
  	<span style="mso-spacerun: yes">&nbsp;&nbsp; </span>otherwise ,
  backbone_intra3 ! [] , <br>
  <span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>RTK_REGULATORY_SITE(res1) &gt;&gt; .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0503.htm');">RTK Intracellular Tyr
Phosphorylation Sites - Option I</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>RTK_SH_BS(res,side_res)::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra2 ! {sh2_tyr} ,
  sh2_tyr ! {res} ,<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>sh2_tyr ? {resa} ,
  RTK_SH_BS(resa, side_res) ;<br>
  <span style="mso-spacerun: yes">&nbsp;
  </span>res ! {sh2_tyr, sh2_tyr1, backbone_intra2, env, side_res} ,<span
  style="mso-spacerun: yes">&nbsp;&nbsp; </span><br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>&lt;&lt; sh2_tyr1
  ? [] , BOUND_RTK_SH_BS ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>sh2_tyr ? {res1} , RTK_SH_BS(res,res1) &gt;&gt;.<br>
  <br>
  BOUND_RTK_SH_BS:-<br>
  <span style="mso-spacerun: yes">&nbsp; </span>dummy ? [] , true .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0505.htm');">RTK Kinase –
Phosphorylation: Option II</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>RTK_KINASE_SITE::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>CROSS_PHOSPHORYLATE +
  FULL_PHOSPHORYLATE .<br>
  <br>
  CROSS_PHOSPHORYLATE::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra1 ?
  {cross_motif} , cross_motif ! {p_tyr} , <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;
  </span>RTK_KINASE_SITE. <br>
  FULL_PHOSPHORYLATE::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra3 ? [] ,
  ACTIVE_FULL .<br>
  <br>
  ACTIVE_FULL::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra2 ?
  {cross_motif} , cross_motif ! {p_tyr} , <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>ACTIVE_FULL
  ;<br>
  <span style="mso-spacerun: yes">&nbsp;</span>backbone_intra3 ? [] ,
  RTK_KINASE_SITE .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0506.htm');">RTK Kinase – Regulation -
Option II</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>RTK_REGULATORY_SITE(res)::=<br>
  	tyr1162 ? {res1} ,</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2><span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>&lt;&lt; res1 =?= res<span
  style="mso-spacerun: yes">&nbsp; </span>, RTK_REGULATORY_SITE(res1) ;<br>
  	<span style="mso-spacerun: yes">&nbsp;&nbsp; </span>otherwise ,
  backbone_intra3 ! [] , <br>
  <span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>RTK_REGULATORY_SITE(res1) &gt;&gt; .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0507.htm');">RTK Intracellular Tyr
Phosphorylation Sites - Option II</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>RTK_SH_BS(res,side_res)::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra2 ! {sh2_tyr} ,
  sh2_tyr ? {resa} ,<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>RTK_SH_BS(resa, side_res) ;<br>
  <span style="mso-spacerun:
  yes">&nbsp; </span>res ! {sh2_tyr, sh2_tyr1, backbone_intra2, env, side_res}
  ,<span style="mso-spacerun: yes">&nbsp;&nbsp; </span><br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>&lt;&lt; sh2_tyr1
  ? [] , BOUND_RTK_SH_BS ;<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>sh2_tyr ? {res1} , RTK_SH_BS(res,res1) &gt;&gt;.<br>
  <br>
  BOUND_RTK_SH_BS:-<br>
  <span style="mso-spacerun: yes">&nbsp; </span>dummy ? [] , true .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0497.htm');">Receptor
<br>
(Trans-phosphorylation)</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0508.htm');">Receptor
<br>
(Trans-phosphorylation)</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0509.htm');">Receptor
<br>
(Trans-phosphorylation)</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0498.htm');">Receptor
<br>
(Auto-phosphorylation)</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0486.htm');">The activated receptor</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The phosphorylated tyrosines can be
  specifically identified by SH2 and SH3 domains on other proteins, including
  adapter proteins</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The activated receptor can then
  phosphorylate these bound proteins</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0466.htm');">Adapter proteins:
<br>
Coupling receptor and Ras activation</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0510.htm');">Binding SH2 domains</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The SH2 domain is a compact module</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Each SH2 domain has distinct sites for
  recognizing phosphotyrosine and for recognizing a particular amino acid side
  chain</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Thus, different SH2 domains recognize
  pTyr in the context of different flanking amino acids</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0511.htm');">Simultaneous recognition
in multiple sites</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Correct identification of an SH2 domain
  requires matching of two motifs (global channels)</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>One approach is to combine
  communication with the match construct</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Alternatively, we may treat each
  combined tyr+flanking region as an independent motif.</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>In this case the phosphorylating kinase
  should modify a more “specific” name</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0512.htm');">RTK Intracellular
Tyrosine Phosphorylation Sites</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>RTK_SH_BS(res,side_res)::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone_intra2 ! {sh2_tyr} ,
  sh2_tyr ? {resa} ,<br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>RTK_SH_BS(resa, side_res) ;<br>
  <span style="mso-spacerun:
  yes">&nbsp; </span>res ! {sh2_tyr, sh2_tyr1, backbone_intra2, mem, side_res}
  ,<span style="mso-spacerun: yes">&nbsp;&nbsp; </span><br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>&lt;&lt; sh2_tyr1
  ? [] , BOUND_RTK_SH_BS ;<br>
  <span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>sh2_tyr ? {res1} ,
  RTK_SH_BS(res,res1) &gt;&gt;.<br>
  <br>
  BOUND_RTK_SH_BS:-<br>
  <span style="mso-spacerun: yes">&nbsp; </span>dummy ? [] | true .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0513.htm');">SHC</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>SHC(env)::=</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>&lt;&lt; shc_tyr, shc_tyr1, shc_tyr2,
  backbone . <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>SHC_SH2(env) |
  SHC_SH2_BS(env,tyr,glu)<span style="mso-spacerun: yes">&nbsp; </span>.<br>
  <br>
  SHC_SH2(env)::=</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2><span style="mso-spacerun:
  yes">&nbsp;</span>p_tyr ? {c_sh2,c_sh2a,c_backbone,c_env, c_res1} , <br>
  <span style="mso-spacerun: yes">&nbsp; </span>&lt;&lt; c_res1 =?= met , backbone
  ! {c_env} , c_backbone!{shc_tyr},<span style="mso-spacerun: yes">&nbsp;&nbsp;
  </span><br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
  </span>c_sh2a ! [] , BOUND_SHC_SH2(c_env) ;<br>
  <span style="mso-spacerun:
  yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>otherwise , c_sh2 ! {c_res1} ,
  SHC_SH2(env) &gt;&gt; .<br>
  	<br>
  BOUND_SHC_SH2(cross_env)::= <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>dummy ? [] ,
  true .</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0514.htm');">SHC</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>SHC_SH2_BS(env,res,res1)::=<br>
  <span style="mso-spacerun: yes">&nbsp; </span>backbone
  ? {cross_env} , SHC_SH2_BS(cross_env,res,res1);<br>
  <span style="mso-spacerun: yes">&nbsp; </span>shc_tyr
  ? {resa} , SHC_SH2_BS(env,resa,res1);<br>
  <span style="mso-spacerun: yes">&nbsp; </span>res1<span style="mso-spacerun:
  yes">&nbsp; </span>! {shc_tyr1, shc_tyr2, env, res} , <br>
  <span style="mso-spacerun: yes">&nbsp;&nbsp;&nbsp; </span>&lt;&lt; shc_tyr1
  ? [] , BOUND_SHC_SH2_BS ;<br>
  	shc_tyr2 ? [] , SHC_SH2_BS(env,res,res1)
  &gt;&gt; .<br>
  <br>
  BOUND_SHC_SH2_BS:-<br>
  <span style="mso-spacerun: yes">&nbsp; </span>dummy ? [] , true &gt;&gt; .</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0515.htm');">SHC binding to receptor
<br>
pTyr-met motif</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0518.htm');">SHC binding to receptor
<br>
pTyr-met motif</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0517.htm');">SHC phosphorylation by
RTK</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0487.htm');">Ras Activation</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>By these protein-protein interactions,
  the SOS protein is brought close to the membrane, where is can activate Ras,
  that is attached to the membrane</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>SOS activates Ras by exchanging Ras’s
  GDP with GTP.</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>GAP inactivates it by the reverse
  reaction</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0467.htm');">Activation of the MAPK
cascade</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Active Ras interacts with the first
  kinase in the MAPK cancade, Raf.</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>It localizes Raf to the membrane, where
  it is activated by an unknown mechanism</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>This starts the cascade</font></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0490.htm');">Activation of the MAPK
cascade</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Each kinase in the cascade is activated
  by phosphorylation in<span style="mso-spacerun: yes">&nbsp; </span>a
  regulatory site, called the t-loop</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>When T-loop is phosphorylated, a
  conformation change occurs and the catalytic cleft is “opened” and active</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>Each kinase is bound by modifying
  enzymes (incoming signals) on its Nt lobe. It binds its substrate through its
  Ct lobe.</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The three kinases may be tethered
  together in one complex with the MP1 scaffold protein</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0489.htm');">Slide 60</a></b></font></p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0468.htm');">MAPK targets</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>The MAPK phosphorylates and activates
  many different targets</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>For example, after phosphorylation it
  may translocate to the nucleus and activate transcription factors</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1><font face=Arial size=2>It also phosphorylates the receptor
  kinase and other enzymes in the pathway in an inhibitory fashion (negative
  feedback)</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=1></td>
 </tr>
</table>

</p>

<p><font face=Arial size=2><b><a
href="javascript:parent.GoToSld('v3_slide0410.htm');">References</a></b></font>

<table>
 <tr>
  <td width=5 nowrap></td>
  <td width=20 nowrap></td>
  <td width="100%"></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>General Introduction:</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Alberts et al. (1994) Molecular Biology
  of the Cell, Chapter 15</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Alberts et al. (1997) Essential Cell
  Biology, Chapter 15</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>Signal transduction:</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Krauss (2000) Biochemistry of Signal
  Transduction and Regulation</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Heldin and Purton (eds.) (1996) Signal
  Transduction</font></td>
 </tr>
 <tr>
  <td colspan=1></td>
  <td colspan=2><font face=Arial size=2>RTK-MAPK pathways</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Lewis et al. (1998) Signal transduction
  through MAP kinase cascades. Advances in Cancer Research 74: 49-139</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Widmann et al (1999) Mitogen activated
  protein kinase: Conservation of a three kinase module from yeast to human.
  Physiological Reviews 79:143-180</font></td>
 </tr>
 <tr>
  <td colspan=2></td>
  <td colspan=1><font face=Arial size=2>Brunet et al (1997) Mammalian MAP
  kinase modules: how to transduce specific signals. Essays in Biochemistry 32:
  1-16.</font></td>
 </tr>
</table>

</p>

</body>

</html>
