<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 9">
<meta name=Originator content="Microsoft Word 9">
<link rel=File-List href="./STDB_links_files/filelist.xml">
<link rel=Edit-Time-Data href="./STDB_links_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]-->
<title>Signal transduction databases</title>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Shmulik Elmakias</o:Author>
  <o:LastAuthor>qbi</o:LastAuthor>
  <o:Revision>28</o:Revision>
  <o:TotalTime>61</o:TotalTime>
  <o:Created>2000-10-06T13:54:00Z</o:Created>
  <o:LastSaved>2001-11-18T18:35:00Z</o:LastSaved>
  <o:Pages>1</o:Pages>
  <o:Words>502</o:Words>
  <o:Characters>2867</o:Characters>
  <o:Company>Q.B.I.</o:Company>
  <o:Lines>23</o:Lines>
  <o:Paragraphs>5</o:Paragraphs>
  <o:CharactersWithSpaces>3520</o:CharactersWithSpaces>
  <o:Version>9.2720</o:Version>
 </o:DocumentProperties>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:"Lucida Sans Unicode";
	panose-1:2 11 6 2 3 5 4 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-2147476737 14699 0 0 63 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
a:link, span.MsoHyperlink
	{color:#FF6600;
	mso-text-animation:none;
	font-weight:bold;
	text-decoration:none;
	text-underline:none;
	text-decoration:none;
	text-line-through:none;}
a:visited, span.MsoHyperlinkFollowed
	{color:red;
	mso-text-animation:none;
	font-weight:bold;
	text-decoration:none;
	text-underline:none;
	text-decoration:none;
	text-line-through:none;}
p
	{margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
p.Avivweb, li.Avivweb, div.Avivweb
	{mso-style-name:Aviv_web;
	mso-style-parent:"Normal \(Web\)";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Lucida Sans Unicode";
	mso-fareast-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";}
@page Section1
	{size:612.0pt 792.0pt;
	margin:72.0pt 90.0pt 72.0pt 90.0pt;
	mso-header-margin:36.0pt;
	mso-footer-margin:36.0pt;
	mso-paper-source:0;}
div.Section1
	{page:Section1;}
-->
</style>
<!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1048"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-US link="#ff6600" vlink=red style='tab-interval:36.0pt'>

<div class=Section1>

<p class=Avivweb align=center style='text-align:center'><span style='font-size:
11.0pt;mso-bidi-font-size:9.0pt'>SIGNAL TRANSDUCTION AND RELATED DATABASES</span><span
style='mso-bidi-font-size:10.0pt'><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="_x0000_i1025" type="#_x0000_t75" style='width:11.25pt;
 height:8.25pt' o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1025"><![endif]><span
style='mso-tab-count:1'>  </span><a
href="http://nar.oupjournals.org/cgi/content/full/28/1/1/DC1" target="_blank"><b>The
Nucleic Acid Research database issue 2000</b></a> provides an updated
compilation of links to all bioinformatics on-line resources, sorted by
category, with concise descriptions. <o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1026" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1026"><![endif]><span
style='mso-tab-count:1'>  </span><a
href="http://www.genome.ad.jp/kegg/regulation.html" target="_blank"><b>KEGG
regulatory pathways</b></a> include Graphical pathway maps, ortholog group
tables, and molecular catalogs for different pathways from various organisms.<o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1027" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1027"><![endif]><span
style='mso-tab-count:1'>  </span><a href="http://transpath.gbf.de/"><b>TRANSPATH</b></a>
<span style='color:black'>provides access to a growing amount of signal transduction
data, mainly to pathways involved in mammalian transcriptional regulation.
Entries are validated with references to original publications and linked to
other databases. The knowledge base goes beyond the approach of traditional
protein databases by storing interactions. <o:p></o:p></span></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span><span style='color:black'><o:p></o:p></span></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1028" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1028"><![endif]><span
style='mso-tab-count:1'>  </span></span><span style='font-size:10.0pt;
mso-bidi-font-size:12.0pt;mso-bidi-font-family:"Lucida Sans Unicode";
color:black'><a href="http://www.ebi.ac.uk/research/pfmp" target="_blank"><b>aMAZE</b></a>,
part of the </span><span style='font-size:10.0pt;mso-bidi-font-size:36.0pt;
mso-bidi-font-family:"Lucida Sans Unicode"'>Protein Function and Biochemical
Pathways (PFBP) Project at the EBI, is an </span><span style='font-size:10.0pt;
mso-bidi-font-size:12.0pt;mso-bidi-font-family:"Lucida Sans Unicode"'>integrated
database encompassing metabolic, regulatory and signal transduction pathways.
As part of the project, a series of specialised querying tools, is being
developed, in particular pathway queries, finding, comparing and abstracting
pathways.<o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span></span><span style='font-size:10.0pt;mso-bidi-font-size:12.0pt;
mso-bidi-font-family:"Lucida Sans Unicode"'><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1029" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1029"><![endif]><span
style='mso-tab-count:1'>  </span></span><span style='font-size:10.0pt;
mso-bidi-font-size:12.0pt;mso-bidi-font-family:"Lucida Sans Unicode"'><a
href="http://www.grt.kyushu-u.ac.jp/spad"><b>SPAD</b></a> compiles graphical clickable
maps of various eukaryotic pathways, categorized according to the extracellular
molecules, which initiate the signal. Links to DNA, RNA and protein sequence
are provided.<o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span></span><span style='font-size:10.0pt;mso-bidi-font-size:12.0pt;
mso-bidi-font-family:"Lucida Sans Unicode"'><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1030" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1030"><![endif]><span
style='mso-tab-count:1'>  </span></span><span style='font-size:10.0pt;
mso-bidi-font-size:12.0pt;mso-bidi-font-family:"Lucida Sans Unicode"'><a
href="http://geo.nihs.go.jp/csndb/paper.html"><b>CSNDB</b></a> </span><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'>is a data- and knowledge-
base for signaling pathways of human cells. It compiles the information on
biological molecules, sequences, structures, functions, and biological
reactions which transfer the cellular signals. Signaling pathways are compiled
as binary relationships of biomolecules and represented by graphs drawn
automatically.<o:p></o:p></span></p>

<p class=Avivweb><span style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1031" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1031"><![endif]><span
style='mso-tab-count:1'>  </span><a href="http://dip.doe-mbi.ucla.edu/"
target="_blank"><b>DIP</b></a>, database of interacting proteins, </span><span
style='font-size:10.0pt'>catalogs experimentally determined interactions
between proteins. It combines information from a variety of sources to create a
single, consistent set of protein-protein interactions.</span><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>    </span><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1032" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1032"><![endif]><span
style='mso-tab-count:1'>  </span><a
href="http://www.mips.biochem.mpg.de/proj/yeast/pathways/index.html"><b>Yeast
Pathways in MIPS</b></a> are given as clickable figures, based on published papers,
with detailed information on sequence, gentetics, protein motifs, transcription
and literature available for each node.<o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span><span style="mso-spacerun: yes"> </span><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1033" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1033"><![endif]><span
style='mso-tab-count:1'>  </span><a
href="http://www.csa.ru/Inst/gorb_dep/inbios/genet/genet.htm"><b>GeNet</b></a>
contains information on the functional organization of regulatory genes
networks acting at embryogenesis. Gene networks are described in interactive
graphical form, based on a Boolean network model. In addition to browse and
search capabilities, genetic networks dynamics can be simulated as well. Cross-organism
links are provided, on the basis of a comparative evolutionary approach.<o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1034" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1034"><![endif]><span
style='mso-tab-count:1'>  </span><a href="http://gifts.univ-mrs.fr/index.html"
target="_blank"><b>GIF_DB</b></a>, <a
href="http://gifts.univ-mrs.fr/FlyNets/FlyNets_home_page.html" target="_blank"><b>FlyNets</b></a>
and <a href="http://sdb.bio.purdue.edu/fly/aimain/1aahome.htm"><b>Interactive
Fly</b></a>, all provide data on transduction and other biochemical pathways in
Drosophila <o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1035" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./STDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./STDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1035"><![endif]><span
style='mso-tab-count:1'>  </span><a href="http://pronet.doubletwist.com/"
target="_blank"><b>ProNet Online</b></a>, Protein-protein interaction database.
</span><span style='font-size:10.0pt;mso-bidi-font-size:12.0pt;mso-bidi-font-family:
"Lucida Sans Unicode"'><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></p>

<p class=Avivweb><span style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></p>

<!---
<LI><A HREF="http://www.iephb.ru/~spirov/genmorph.html"><I>Drosophila</I> Genes in Development</A>
---></div>

</body>

</html>
