<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 9">
<meta name=Originator content="Microsoft Word 9">
<link rel=File-List href="./metDB_links_files/filelist.xml">
<link rel=Edit-Time-Data href="./metDB_links_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]-->
<title>Metabolic pathways databases</title>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Shmulik Elmakias</o:Author>
  <o:LastAuthor>qbi</o:LastAuthor>
  <o:Revision>25</o:Revision>
  <o:TotalTime>70</o:TotalTime>
  <o:Created>2000-10-06T13:55:00Z</o:Created>
  <o:LastSaved>2001-11-18T18:35:00Z</o:LastSaved>
  <o:Pages>1</o:Pages>
  <o:Words>316</o:Words>
  <o:Characters>1802</o:Characters>
  <o:Company>Q.B.I.</o:Company>
  <o:Lines>15</o:Lines>
  <o:Paragraphs>3</o:Paragraphs>
  <o:CharactersWithSpaces>2212</o:CharactersWithSpaces>
  <o:Version>9.2720</o:Version>
 </o:DocumentProperties>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
@font-face
	{font-family:"Lucida Sans Unicode";
	panose-1:2 11 6 2 3 5 4 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-2147476737 14699 0 0 63 0;}
 /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
a:link, span.MsoHyperlink
	{color:#FF6600;
	mso-text-animation:none;
	font-weight:bold;
	text-decoration:none;
	text-underline:none;
	text-decoration:none;
	text-line-through:none;}
a:visited, span.MsoHyperlinkFollowed
	{color:red;
	mso-text-animation:none;
	font-weight:bold;
	text-decoration:none;
	text-underline:none;
	text-decoration:none;
	text-line-through:none;}
p
	{margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
p.Avivweb, li.Avivweb, div.Avivweb
	{mso-style-name:Aviv_web;
	mso-style-parent:"Normal \(Web\)";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Lucida Sans Unicode";
	mso-fareast-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";}
@page Section1
	{size:612.0pt 792.0pt;
	margin:72.0pt 90.0pt 72.0pt 90.0pt;
	mso-header-margin:36.0pt;
	mso-footer-margin:36.0pt;
	mso-paper-source:0;}
div.Section1
	{page:Section1;}
-->
</style>
<!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1033"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-US link="#ff6600" vlink=red style='tab-interval:36.0pt'>

<div class=Section1>

<p class=Avivweb align=center style='text-align:center'><span style='font-size:
11.0pt;mso-bidi-font-size:9.0pt'>METABOLIC PATHWAYS DATABASES</span><span
style='font-size:14.0pt;mso-bidi-font-size:12.0pt'><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><![if !supportEmptyParas]>&nbsp;<![endif]><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="_x0000_i1025" type="#_x0000_t75" style='width:11.25pt;
 height:8.25pt' o:bullet="t">
 <v:imagedata src="./metDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img width=15 height=11
src="./metDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1025"><![endif]><span
style='mso-tab-count:1'>  </span><a href="http://www.brenda.uni-koeln.de/"
target="_blank"><b>BRENDA</b></a>, the enzyme database, has comprehensive
information on enzymes and enzymatic reactions. It is one of several databases
nested within the metabolic pathway database set of the</span><strong><span
style='font-size:10.0pt;font-weight:normal'> </span></strong><strong><span
style='font-size:10.0pt'><a href="http://srs.ebi.ac.uk/" target="_blank">SRS5
sequence retreival system</a></span></strong><strong><span style='font-size:
10.0pt;font-weight:normal'> </span></strong><span style='font-size:10.0pt;
mso-bidi-font-size:12.0pt'>at EBI.&nbsp; <o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1026" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./metDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./metDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1026"><![endif]><span
style='mso-tab-count:1'>  </span><a
href="http://www.genome.ad.jp/kegg/metabolism.html" target="_blank"><b>KEGG
Metabolic Pathways</b></a> include graphical pathway maps for all known
metabolic pathways from various organisms. Ortholog group tables, containing
conserved, functional units in a molecular pathway or assembly as well
comparative lists of genes for a given functional unit in different organisms,
are also available.<o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1027" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./metDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./metDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1027"><![endif]><span
style='mso-tab-count:1'>  </span><a href="http://wit.mcs.anl.gov/WIT2/"
target="_blank"><b>The WIT Metabolic Reconstruction project</b></a> produces
metabolic reconstructions for sequenced, or partially sequenced, genomes. It
currently provides a set of over 25 such reconstructions in varying states of
completion. Over 2900 pathway diagrams are available, associated with
functional roles and linked to ORFs. <o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1028" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./metDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./metDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1028"><![endif]><span
style='mso-tab-count:1'>  </span><a
href="http://ecocyc.pangeasystems.com/ecocyc/ecocyc.html" target="_blank"><b>EcoCyc</b></a>
describes the genome and the biochemical machinery of E. coli. It provides a
molecular and functional catalog of the <i>E. coli</i> cell to facilitates
system-level understanding. Its Pathway/Genome Navigator user interface visualizes
the layout of genes, of individual biochemical reactions, or of complete
pathways. It also supports computational studies of the metabolism, such as
pathway design, evolutionary studies, and simulations. A related metabolic
database is <a href="http://indigo.genetique.uvsq.fr/" target="_blank"><b>Metalgen</b></a>.<o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><span style='mso-tab-count:
1'>      </span><o:p></o:p></span></p>

<p class=Avivweb style='margin-left:36.0pt;text-indent:-18.0pt;tab-stops:list 36.0pt'><span
style='font-size:10.0pt;mso-bidi-font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1029" type="#_x0000_t75" style='width:11.25pt;height:8.25pt'
 o:bullet="t">
 <v:imagedata src="./metDB_links_files/image001.gif" o:title="bd21299_"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=15 height=11
src="./metDB_links_files/image001.gif" alt="*" v:shapes="_x0000_i1029"><![endif]><span
style='mso-tab-count:1'>  </span><a
href="http://www.expasy.ch/cgi-bin/search-biochem-index" target="_blank"><b>Boehringer
Mannheim - Biochemical Pathways</b></a> is a searchable database of metabolic
pathways, enzymes, substrates and products. Based on a given search, it
produces a graphic representation of the relevant pathway(s) within the context
of an enormous metabolic map. Neighboring metabolic reactions can then be
viewed through links to adjacent maps. <o:p></o:p></span></p>

</div>

</body>

</html>
